src/Microsoft.AspNetCore.Http.Abstractions/HostString.cs

```diff
@@ -38,6 +64,92 @@ public bool HasValue
         }
 
         /// <summary>
+        /// Returns the value of the domain part of the host. The port is removed if it was present.
+        /// IPv6 addresses will have brackets added if they are missing.
+        /// </summary>
+        /// <returns></returns>
+        public string Domain
```

Comment:
Officially it's called the Host, not the Domain. https://www.w3.org/Protocols/rfc2616/rfc2616-sec14.html#sec14.23